common-summoning.lic

```diff
@@ -72,22 +68,24 @@ module DRCS
     waitrt?
   end
 
-  def moon_used_to_summon_weapon
```

Comment:
Per the dependencies listed in the description, this method has already moved to common-moonmage